Reqnroll/EnvironmentAccess/EnvironmentOptions.cs

```diff
@@ -13,11 +14,22 @@ public class EnvironmentOptions(IEnvironmentWrapper environment) : IEnvironmentO
     public const string REQNROLL_BINDING_OUTPUT_ENVIRONMENT_VARIABLE = "REQNROLL_BINDING_OUTPUT";
     public const string DOTNET_RUNNING_IN_CONTAINER_ENVIRONMENT_VARIABLE = "DOTNET_RUNNING_IN_CONTAINER";
 
-    public bool IsDryRun => 
+    private readonly Lazy<bool> _isDryRunLazy;
```

Comment:
Here's a suggestion for simplifying this to avoid a few extra private members: #878

Feel free to pull into your branch if desired!